Majasdarbs/02-majasdarbs/Portfolio/default_template.html

```diff
@@ -1,29 +1,105 @@
         <address>
-          <h1>Contact info:</h1>
+          <h1>My contact info:</h1>
+          <p>Phone number:+371 22011589</p>
+          <p>E-mail: enni.zuka@inbox.lv</p>
+          <p>Github: @ZukaEnni</p>
```

Comment:
Either add link to the github (<a href="link"></a>) or replace this with linkedin, which is more reasonable to include in cv.